test/read.js

```diff
@@ -52,7 +167,7 @@ describe("Testing jsonapi-client", function() {
       var somePhoto = person.photos[0];
       assert.throws(function() {
         somePhoto.relationships("photographer").add("foobar");
-      }, /Expected Resource, got String/);
+      }, /Expected Resource, got /);
```

Comment:
This was bugging me and I downloaded a IE9 VM to test this in the console and I got `typeof "foobar" === ?string"`: are you sure this isn't a symptom of some other issue in these browsers? Probably not worth the effort of investigating though.